lib/sles4sap_publiccloud.pm

```diff
@@ -1240,9 +1240,12 @@ sub pacemaker_version {
 
 sub saphanasr_showAttr_version {
     my ($self) = @_;
-    my $version_cmd = 'SAPHanaSR-showAttr --version';
 
-    my $version_output = $self->run_cmd(cmd => $version_cmd, quiet => 1);
+    my $which_output = $self->run_cmd(cmd => 'which SAPHanaSR-showAttr', quiet => 1);
+    record_info('which SAPHanaSR-showAttr', $which_output);
```

Comment:
Why printing it at each execution is important? What is the information that it gives?

Reply:
One is `/usr/bin/SAPHanaSR-showAttr` the other is `/usr/sbin/SAPHanaSR-showAttr`, the path  are different so I think it could be better to print it.

Reply:
If we need it we can maybe have a single `record_info` and print both `$which_output` and `$version_output` at line 1248

Reply:
> If we need it we can maybe have a single `record_info` and print both `$which_output` and `$version_output` at line 1248

Good idea, done!